jsonnet/.gitignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
it should be sufficient to have the entry in the .gitignore at the root, no?

Reply:
Definitely, I've forgotten to clean things up